index.html

```diff
@@ -14,9 +14,9 @@
     <input type="text" name="Email" value="Your email address...">
     <button type="button">Notify Me</button>
     <div class="socialMedia">
-        <i class="fa fa-twitter-square fa_custom fa-4x"></i>
-        <i class="fa fa-facebook-square fa_custom fa-4x"></i>
-        <i class="fa fa-instagram fa_custom fa-4x"></i>
+        <a href="https://twitter.com/TheBlackBinder?lang=en-gb" target="_blank"><i class="fa fa-twitter-square fa_custom fa-4x"></i></a>
+        <a href="https://m.facebook.com/The-Black-Binder-1865704627005645/?tsid&refid=46" target="_blank"><i class="fa fa-facebook-square fa_custom fa-4x"></i></a>
```

Comment:
You are sending people to the mobile site.

What is the query at the end of `facebook` and `twitter` ?